src/lib/load-geojson.js

```diff
@@ -13,7 +21,58 @@ const requestOptions = ({ namespace, layer }) => ({
   maxFeatures: 3000
 })
 
-export async function loadGeojson (jsonLayer, { filteredIds = [] } = {}) {
+const getFeatures = (url, jsonLayer, layerSetId, retries = 0) => {
+  return new Promise((resolve, reject) => {
+    fetch(url, { mode: 'cors' })
+      .then(resp => {
+        // Workaround for a bug in the geoserver,
+        // it returns 200 but the response is an XML with an error message
+        if (
+          resp.headers.get('content-type') &&
+          !resp.headers.get('content-type').includes('application/json')
+        ) {
+          caches.delete(url)
+          throw new Error('Response is not JSON')
+        }
+        return resp.json()
+      })
+      .then(geojson => {
+        geojson.features = geojson.features.map(feature => {
+          feature.properties.isControllable = !!jsonLayer.iscontrollayer
+          feature.properties.icon = 'default'
+          return feature
+        })
+
+        resolve(geojson)
+      })
+      .catch(error => {
+        if (retries < MAX_RETRIES) {
+          console.warn(`Retry ${retries + 1} - ${error}`)
+          setTimeout(
+            () =>
+              getFeatures(url, jsonLayer, layerSetId, retries + 1)
+                .then(resolve)
+                .catch(reject),
+            RETRY_DELAY
+          )
+        } else {
+          console.error(`Max retries exceeded - ${error}`)
+          // Show an error notification if the layer is not available
+          const notification = {
+            message: 'Please retry in 5 to 10 minutes.',
```

Comment:
I need a translation for this message, but I don't know how much we want to explain about the real reason why this is failing.